flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -120,6 +120,11 @@ class SentryFlutterOptions extends SentryOptions {
   /// See https://api.flutter.dev/flutter/foundation/FlutterErrorDetails/silent.html
   bool reportSilentFlutterErrors = false;
 
+  /// Enables Out of Memory Tracking for iOS and macCatalyst.
+  /// See the following link for more information and possible restrictions:
+  /// https://docs.sentry.io/platforms/apple/guides/ios/configuration/out-of-memory/
+  bool enableAppleOutOfMemoryTracking = false;
```

Comment:
I'd just call `enableOutOfMemoryTracking` though and document is iOS only, if the feature we somehow manage to support in more platforms, we don't have to rename nor introduce a new flag